### PR TITLE
DP-27215: Dependabot - Multiple vulnerabilities

### PR DESCRIPTION
**Pull request by arthurbaghdas**
**Description:**
Explain the technical implementation of the work done.

**Jira:** (Skip unless you are MA staff)
DP-27215

**To Test:**
- [ ] Add steps to test this feature

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docroot/sites/default/default.settings.php [hunk 1]
```diff
@@ -138,6 +138,21 @@
...
+ * in deadlocks, the other two options are 'READ UNCOMMITTED' and 'SERIALIZABLE'.
+ * They are available but not supported; use them at your own risk. For more
+ * info:
+ * https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html
+ *
+ * On your settings.php, change the isolation level:
+ * @code
+ * $databases['default']['default']['init_commands'] = [
+ *   'isolation_level' => 'SET SESSION TRANSACTION ISOLATION LEVEL READ COMMITTED',
+ * ];
```
Comment (weitzman) [inline docroot/sites/default/default.settings.php:153]: We need to add this to our database declarations in settings.php. I think you could do that just above "// Add database connection for Service Details migration.". I consider this change a part of adopting Drupal 9.5 which is what this PR is doing.

File: docroot/sites/default/default.settings.php [hunk 2]
```diff
@@ -641,33 +644,6 @@
...
- *   match paths that should return a simple 404 page, rather than the fully
- *   themed 404 page. If you don't have any aliases ending in htm or html you
- *   can add '|s?html?' to the expression.
- * - $config['system.performance']['fast_404']['html']: The html to return for
- *   simple 404 pages.
- *
- * Remove the leading hash signs if you would like to alter this functionality.
- */
-# $config['system.performance']['fast_404']['exclude_paths'] = '/\/(?:styles)|(?:system\/files)\//';
-# $config['system.performance']['fast_404']['paths'] = '/\.(?:txt|png|gif|jpe?g|css|js|ico|swf|flv|cgi|bat|pl|dll|exe|asp)$/i';
```
Comment (weitzman) [inline docroot/sites/default/default.settings.php:668]: Lets remove the fast_404 lines that are active in our settings.php

**Review by weitzman — CHANGES_REQUESTED** [review]
Looks good. Just a couple minor changes needed.

**Comment by weitzman** [comment]
Also, did you do a `drush cex` after the upgrade? Sometimes there are config changes that we add to the PR.

File: docroot/sites/default/default.settings.php (hunk 1 above)
Comment (arthurbaghdas) [reply docroot/sites/default/default.settings.php:153]: Added

File: docroot/sites/default/default.settings.php (hunk 2 above)
Comment (arthurbaghdas) [reply docroot/sites/default/default.settings.php:668]: Removed

**Comment by arthurbaghdas** [comment]
> Also, did you do a `drush cex` after the upgrade? Sometimes there are config changes that we add to the PR.

Yes, no config exports or update hooks available.

**Pull request merged**